static/app/views/settings/organizationGeneralSettings/organizationSettingsForm.tsx

```diff
@@ -21,9 +22,21 @@ type Props = {
 
 type State = AsyncComponent['state'] & {
   authProvider: object;
+  orgSettingsFields: JsonFormObject[];
 };
 
 class OrganizationSettingsForm extends AsyncComponent<Props, State> {
+  componentWillMount() {
+    const idempotencyKey = [...Array(2)]
+      .map(_ => Math.random().toString(36).substr(2, 10))
+      .join('');
```

Comment:
You should be able to set this value to the class during willMount and then splice it into the `initialData` during render.

```typescript
private idempotency: string = '';

componentWillMount() {
  this.idempotency = [...Array(2)]
      .map(_ => Math.random().toString(36).substr(2, 10))
      .join('');
}
```

And then in render

```typescript
    const {initialData, organization, onSave, access} = this.props;
    const initial = {...initialData, idempotencyKey: this.idempotency}
```

And use `initial` for the form's `initialData` prop.